### PR TITLE
WIP: Demo Sources and Release Helper

### PR DESCRIPTION
**Pull request by AndreasBilke**
Move the sources to our main repo and write a helper script to automatize the creating of a pdfpc release.

The `make_release.sh` needs careful testing :) I want to discuss the approach of the script at all.

### CODE REVIEW

**Comment by fnevgeny** [comment]
I suggest making demo zip through a make rule, so there would be no need to "make clean" before and we can list the files to be zip'ed explicitly.

File: make_release.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    zip -j ../pdfpc-video-example.zip video-example.pdf video-example.tex apollo17.avi apollo17.jpg
+    cd ../../
+}
+
+function create_release() {
+    API_URL=$1
+    TAG_NAME=$2
+
+    curl --data <<<EOF
```
Comment (fnevgeny) [inline make_release.sh]: Shouldn't it be "<<" instead of "<<<"?

Comment (AndreasBilke) [reply make_release.sh]: You are right. But using heredoc at this point was wrong anyways :)

**Comment by AndreasBilke** [comment]
Okay, now everything should work. I tested it in my pdfpc clone.

**Pull request merged**